### PR TITLE
cloud: ovirt: Add download image support to disks module

### LINKED ISSUE #22088: ovirt_disks: Download disks functionality.
##### ISSUE TYPE

  - Feature Idea

##### COMPONENT NAME

ovirt_disks

##### ANSIBLE VERSION

```
ansible 2.3.0 (devel 9f49d4b8f4) last updated 2017/02/27 16:51:46 (GMT +300)
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
```

##### SUMMARY

oVirt has the ability to download a VM disk using API. this flow can be useful when a user would like to transfer VM images between different oVirt instances.
please add the ability to download a disk image from oVirt to the Ansible modules.
 
##### STEPS TO REPRODUCE

```yaml

```

##### ACTUAL RESULTS

```

```

### PR DESCRIPTION
**Pull request by machacekondra**
##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME

ovirt_disks

##### ANSIBLE VERSION

```
2.3.0
```

##### SUMMARY

This PR fixes: https://github.com/ansible/ansible/issues/22088

```

```

### CODE REVIEW

**Comment by machacekondra** [comment]
@mwperina

**Comment by mattclay** [comment]
CI failure due to module docs validation error:

```
2017-03-01 01:41:20 ============================================================================
2017-03-01 01:41:20 lib/ansible/modules/cloud/ovirt/ovirt_disks.py
2017-03-01 01:41:20 ============================================================================
2017-03-01 01:41:20 ERROR:309:version_added for new option (download_image_path) should be 2.3. Currently 0.0
```

File: lib/ansible/module_utils/ovirt.py [hunk 1]
```diff
@@ -144,15 +144,6 @@ def create_connection(auth):
         token=auth.get('token', None),
         kerberos=auth.get('kerberos', None),
     )
-    api_version = LooseVersion(engine_version(connection))
-    python_sdk_version = LooseVersion(sdk_version.VERSION)
-    if python_sdk_version < api_version:
```
Comment (machacekondra) [inline lib/ansible/module_utils/ovirt.py:149]: This is solved by SDK now.

**Review by mwperina — APPROVED** [review]
👍

**Comment by machacekondra** [comment]
@ryansb shipit

**Pull request merged**